drivers/video/Kconfig

```diff
@@ -25,6 +25,10 @@ config FB_SYNC
 	bool "Hardware signals vertical sync"
 	default n
 
+config FB_DOUBLE_BUFFER
```

Comment:
where do you use FB_DOUBLE_BUFFER? the chip/board specific option should be kept inside Kconfig under arch/board.